src/python/pants/pantsd/pants_daemon.py

```diff
@@ -165,6 +165,8 @@ def __init__(self, native, build_root, work_dir, log_level, services, socket_map
     self._logger = logging.getLogger(__name__)
     # A lock to guard the service thread lifecycles.
```

Comment:
Larger docstrings on both of these (which other pydocs can refer to by name) would be awesome... "the ${abc}_lock guards X, Y, Z" etc.